README.md

```diff
@@ -1,3 +1,81 @@
 # regnskapsregister-API
 
 Nøkkeltall fra årsregnskapet for siste regnskapsår som åpne data.
+
+# Local development
```

Comment:
Skal regnskapsregister-api dokumenteres på norsk eller engelsk? Spør for en venn.

Reply:
Dunno, andre ( kremt @stigbd ) skriver på norsk. Kan fint oversette, men norsk -> engelsk eller engelsk -> norsk?

Reply:
8e9fe271024c90558401400e13618777fbad0633 Ok oversettelse?